sdks/builds/mxe.mk

```diff
@@ -1,31 +1,19 @@
 
 MXE_SRC?=$(TOP)/sdks/builds/toolchains/mxe
-MXE_PREFIX?=$(TOP)/sdks/out/mxe
+MXE_PREFIX_DIR?=$(TOP)/sdks/out
 
-$(TOP)/sdks/builds/toolchains/mxe:
-	git clone -b xamarin https://github.com/xamarin/mxe.git $@
-	cd $@ && git checkout $(MXE_HASH)
+# This is not overridable
+MXE_PREFIX=$(MXE_PREFIX_DIR)/mxe-$(shell echo $(MXE_HASH) | head -c 6)
```

Comment:
I don't think we really need to shorten MXE_HASH here, the full sha is OK.

At the very least we should be using `:=` for variable assignment so we don't shell out each time this variable is used.

Reply:
Shortening `MXE_HASH` is the approach taken by XA, so for the sake of ease of integration, I didn't want to do it differently.